src/cljam/io/vcf/util.clj

```diff
@@ -10,13 +10,22 @@
        (and (= 1 (.length ~s))
             (= \. (.charAt ~s 0)))))
 
+(defn- parse-float [s]
+  (let [m (re-matches
+           #"[+-]?(?i)(nan|infinity|inf)" s)]
+    (if m
+      (case (first (second m))
+        (\n \N) Float/NaN
+        (\i \I) Float/POSITIVE_INFINITY)
```

Comment:
Should `-inf` and `-infinity` return `Float/NEGATIVE_INFINITY`?

Reply:
You are right.
Thank you.